test/remote/gateways/remote_plexo_test.rb

```diff
@@ -158,4 +158,103 @@ def test_transcript_scrubbing
     assert_scrubbed(@credit_card.number, transcript)
     assert_scrubbed(@gateway.options[:api_key], transcript)
   end
+
+  def test_successful_purchase_passcard
+    credit_card = credit_card('6280260025383009', month: '12', year: '2024',
+      verification_value: '111', first_name: 'Santiago', last_name: 'Navatta')
+
+    response = @gateway.purchase(@amount, credit_card, @options)
+    assert_success response
+  end
+
+  def test_successful_purchase_edenred
+    credit_card = credit_card('6374830000000823', month: '12', year: '2024',
+      verification_value: '111', first_name: 'Santiago', last_name: 'Navatta')
+
+    response = @gateway.purchase(@amount, credit_card, @options)
+    assert_success response
+  end
+
+  def test_successful_purchase_anda
+    credit_card = credit_card('6031991248204901', month: '12', year: '2024',
+      verification_value: '111', first_name: 'Santiago', last_name: 'Navatta')
+
+    response = @gateway.purchase(@amount, credit_card, @options)
+    assert_success response
+  end
+
+  def test_successful_purchase_and_declined_refund_anda
+    omit
```

Comment:
❓ Question:

Is this intentional ?, if it is, may be a comment telling why this test is being skipped is a good call.

Reply:
yeah!! It's intentional. Currently, refund, void transaction is approved not declined. is omitted until plexo confirms that the credit card number would be rejected at refund or void transaction request.